### ISSUE TITLE
PostCSS-Gram functionality

### ISSUE THREAD

**Issue by jonathantneal** [una/postcss-gram#1]
Hi Una!

I saw you added a few new filters this week, and I was wondering if you’d like a PR on this to add them.

At first I thought maybe [PostCSS Instagram](https://github.com/azat-io/postcss-instagram) was the PostCSS edition of your filters, but I don’t see any mention there of your work.

Your Sass extends for cssgrams are so good! If you’re open to it, I’d love for the PostCSS version to work the same way, where any rule that use a cssgram `filter` (or `instafilter` as I see you call it in a test) has its selector added to a master rule that contains the CSS for the filter, that way people can tweak that one master rule and it would be like tweaking that filter for any element that uses it. Does that make sense? Any interest?

**Comment by una** [comment]
Yeah -- this has been something I've been meaning to get around to but then I heard there already was a PostCSS Instagram (like you sent in that link) so I sort of put it on the back burner for now to focus on some other things

I can make you a contributor if you think it'll be a fun project to work on.

**Comment by jonathantneal** [comment]
Sure, I’d love to contribute.

( writing PostCSS Plugins has been like gardening for me, just kind of relaxing after a long day )

**Comment by azat-io** [comment]
It was not her filters in PostCSS Instagram plugin. :smiley:

**Comment by una** [comment]
Yeah they're unrelated -- I think the PostCSS Instagram plugin is SVG based (which is better in many cases, to be fair)

**Comment by azat-io** [comment]
Also [CSSGram](https://github.com/una/CSSgram/) first commit was after PostCSS Instagram release.
Maybe you can send some PR and we can make this plugin better. Because PostCSS Instagram is not looks very good now in my opinion. I want to make it better soon.
